src/claim/Claimer.ts

```diff
@@ -47,16 +47,19 @@ export default class Claimer implements IClaimer {
    *
    * @param mnemonic The mnemonic which is used to generate the key.
    * @param password The password which is used to generate the key.
+   * @param keyLength The key length of the new secret. Note that this secret will only support credentials and attester with the same key length.
    * @returns A new claimer.
    */
   public static async buildFromMnemonic(
     mnemonic: string,
-    password = ''
+    password = '',
+    keyLength = 1024
```

Comment:
Why do we add `keyLength` to the input, if claimer.go line 17 solely the default key length is used?

Reply:
```go
if len(inputs) > 0 && !inputs[0].IsUndefined() {
	keyLength = inputs[0].Int()
}
sysParams, success := gabi.DefaultSystemParameters[keyLength]
```

not sure what you mean. I think the default thing is overwritten if a key length is given.

Reply:
You're right, sorry. Are 2048 and 4096 bit not available for the claimer key length? 
```typescript
keyLength: 1024 | 2048 | 4096 = 1024
```

Reply:
No i simply forgot to add the type there.